pkg/arvo/sys/zuse.hoon

```diff
@@ -7950,6 +7950,60 @@
     ?:  (compare key.n.a key.n.b)
       $(l.b $(b l.b, r.a ~), a r.a)
     $(r.b $(b r.b, l.a ~), a l.a)
+  ::
+  ::  +subset: take a range excluding start and/or end and all elements
+  ::  outside the range
```

Comment:
Are we sure exclusive is most useful here?  Are there cases where we'd want inclusive?  Would be pretty easy to do I think, just don't `nip` in the `=` case.

Reply:
Exclusive is most useful in the case I'm wanting to use it in, in which I keep track of the last event I've received, and ask for all events after that one. I'm not sure if it's always more useful.